src/utils/layout-helpers.ts

```diff
@@ -139,46 +140,90 @@ export function isELSAddonRoot(root: string) {
   return hasEmberLanguageServerExtension(pack);
 }
 
-export function getProjectInRepoAddonsRoots(root: string) {
-  const prefix = isModuleUnificationApp(root) ? 'packages' : 'lib';
-  const addons = safeWalkSync(path.join(root, prefix), {
-    directories: true,
-    globs: ['**/package.json'],
-  });
-  const engineAddons = safeWalkSync(path.join(root, 'engines'), {
-    directories: true,
-    globs: ['**/package.json'],
-  });
-  const roots: string[] = [];
+function getRecursiveInRepoAddonRoots(root: string, roots: string[]) {
+  const packageData = getPackageJSON(root);
+  const emberAddonPaths: string[] = (packageData['ember-addon'] && packageData['ember-addon'].paths) || [];
+
+  if (roots.length) {
+    if (!isEmberAddon(packageData)) {
+      return [];
```

Comment:
should we return roots here?

Reply:
i dont think so, becauseif the packageData is not an addon, then there shouldnt be any roots returned